src/components/rules-list-page/SyncRulesPage.jsx

```diff
@@ -244,6 +291,27 @@ class SyncRulesPage extends React.Component {
 
     changeLastExecutedFilter = moment => this.setState({ lastExecutedFilter: moment });
 
+    onSearchRequest = key =>
+        this.props.d2.Api.getApi()
+            .get("sharing/search", { key })
+            .then(searchResult => searchResult);
```

Comment:
I guess you were planning to do some transformation here?

Reply:
Yeah, will clean it up. Thanks!